src/components/Delegation/DelegationCards.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo, useState } from 'react'
+import Skeleton from 'react-loading-skeleton'
+import 'react-loading-skeleton/dist/skeleton.css'
+
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid/Grid'
+
+import { DelegationResult, DetailedScores } from '../../clients/SnapshotGraphql'
+import { OPEN_CALL_FOR_DELEGATES_LINK } from '../../constants'
+import Empty from '../Common/Empty'
+import FullWidthButton from '../Common/FullWidthButton'
+import Scale from '../Icon/Scale'
+
+import './DelegationCards.css'
+import DelegatorCardProfile from './DelegatorCardProfile'
+import { VotingPowerListModal } from './VotingPowerListModal'
+
+interface Props {
+  delegation: DelegationResult
+  scores: DetailedScores
+  isLoading?: boolean
+  isUserProfile?: boolean
+}
+
+const MAX_DELEGATIONS_SHOWN = 6
+
+function createGroups<T>(array: T[], itemsPerGroup: number) {
+  const groupsAmount = Math.ceil(array.length / itemsPerGroup)
+  return new Array(groupsAmount).fill('').map((_, i) => array.slice(i * itemsPerGroup, (i + 1) * itemsPerGroup))
+}
+
+function DelegationCards({ delegation, scores, isLoading, isUserProfile }: Props) {
+  const t = useFormatMessage()
+  const delegatedFrom = delegation.delegatedFrom
+  const [showAllDelegations, setShowAllDelegations] = useState(false)
+  const delegationsList = useMemo(
+    () =>
+      delegatedFrom && delegatedFrom.length > 0 && Object.keys(scores).length !== 0
+        ? delegatedFrom
+            .map(({ delegator }) => {
+              const delegatorAddress = delegator.toLowerCase()
+              return { delegator, vp: (scores[delegatorAddress] && scores[delegatorAddress].ownVp) || 0 }
+            })
+            .sort((d1, d2) => (d1.vp > d2.vp ? -1 : d1.vp < d2.vp ? 1 : 0))
+        : [],
+    [delegatedFrom, scores]
+  )
+  const delegationsToShow = useMemo(() => delegationsList.slice(0, MAX_DELEGATIONS_SHOWN), [delegationsList])
+  const areThereMoreDelegations = useMemo(() => delegationsList.length > MAX_DELEGATIONS_SHOWN, [delegationsList])
+
+  return (
+    <div className="DelegationCards">
+      {isLoading && (
+        <>
+          <Skeleton height={70} />
+          <br />
+          <Skeleton height={70} />
+        </>
+      )}
+      {!isLoading &&
+        (delegationsToShow.length > 0 ? (
+          <Grid columns={3} stackable>
+            {createGroups(delegationsToShow, 3).map((delegation, idx) => (
```

Comment:
Maybe you can use `<Grid columns="equal">` instead of dividing the initial array in 3 groups.
https://react.semantic-ui.com/collections/grid/#variations-equal-width-column

Reply:
I need to separate the array anyway in order to have 2 different rows, otherwise it looks like this:

<img width="1269" alt="image" src="https://user-images.githubusercontent.com/45410089/190181037-b1cc5458-e0b7-4136-b75e-be0b82287677.png">

Reply:
Right. What about... `<Grid stackable columns={3}>`?